tests/framework/utils.py

```diff
@@ -477,3 +477,42 @@ def wait_process_termination(p_pid):
     except ChildProcessError:
         return
     raise Exception("{} process is still alive: ".format(stdout.strip()))
+
+
+def get_firecracker_version_from_toml():
+    """
+    Return the version of the firecracker crate, from Cargo.toml.
+    Usually different from the output of `./firecracker --version`, if
+    the code has not been released.
+    """
+    cmd = "cd ../src/firecracker && cargo pkgid | cut -d# -f2 | cut -d: -f2"
+
+    rc, stdout, _ = run_cmd(cmd)
+    assert rc == 0
+
+    return stdout
+
+
+def compare_release_versions(first, second):
```

Comment:
Would rename to compare_versions. Being a release or not does not impact the outcome here.

Reply:
done